test/simulator/timestamp/src/include/session_simulator.h

```diff
@@ -69,6 +69,7 @@ class session_simulator {
     void set_durable_timestamp(uint64_t);
     void set_prepare_timestamp(uint64_t);
     int set_read_timestamp(uint64_t);
+    void reset_txn_level_var();
```

Comment:
Can you keep the A-Z order?

Reply:
Instead, I have grouped the _get_, the _has_ and the _is_, and then alphabetized them.